funding/manager.go

```diff
@@ -1550,6 +1550,7 @@ func (f *Manager) fundeeProcessOpenChannel(peer lnpeer.Peer,
 			// Fail the funding flow.
 			flowErr := fmt.Errorf("channel acceptor blocked " +
 				"zero-conf channel negotiation")
+			log.Error(flowErr)
```

Comment:
Since we log this as an error (even though it was user code causing it), should we add some context? For example: `log.Errorf("Cancelling funding flow for %v based on channel acceptor response: %v", cid, err)`?

Might be worth adding to the other logged errors as well.

Reply:
That seems redundant since we already log "Cancelling funding reservation for node_key=%x, chan_id=%x" in cancelReservationCtx which is called by failFundingFlow.

However, I did notice two other error messages on lines 1583 and 1593 that should also be logged IMO.

Reply:
I think we need a way to correlate the logs, so we should add the `cid` here, otherwise don't think it helps much.